web-client/src/presenter/actions/setDefaultTableSortAction.test.js

```diff
@@ -65,41 +66,40 @@ describe('setDefaultTableSortAction', () => {
       },
     });
 
-    expect(state.tableSort.sortField).toEqual('completedAt');
+    expect(state.tableSort.sortField).toEqual('createdAt');
     expect(state.tableSort.sortOrder).toEqual(DESCENDING);
   });
 
-  it('should not set the tableSort if role is not ADC', async () => {
+  it('the completed should be sorted by createdAt DESCENDING for adc user', async () => {
```

Comment:
I think there's a discrepency between the test name and the expectations here, line 73 says the completed messages should be sorted by `createdAt` but the expectation says `completedAt`.

Reply:
Good catch